test/configs/oscap-rhel8.json

```diff
@@ -1,6 +1,20 @@
 {
   "name": "oscap-rhel8",
   "blueprint": {
+    "packages": [
+      {
+        "name": "scap-security-guide"
```

Comment:
So I have a couple of thoughts on this. But this is probably just a nitpick at the end of the day.
If the `oscap` customization is specified, this package should be installed to the image by default.
So, maybe we want to leave this package out and the test will fail if it somehow doesn't get added.
See:
https://osbuild.org/docs/user-guide/oscap-remediation#openscap

Reply:
Thanks.  I'll drop it!

Reply:
Dropped.